miden-tx/src/executor/mod.rs

```diff
@@ -104,12 +104,9 @@ impl TransactionExecutor {
 
     /// Loads the provided code into the internal MAST forest store and adds the commitment of the
     /// provided code to the commitments set.
-    pub fn load_account_code(&mut self, code: &AccountCode) {
-        // load the code mast forest to the mast store
-        self.mast_store.load_account_code(code);
```

Comment:
I would keep this. This way, we don't need to re-load MAST forests for each transaction separately (i.e., on lines 144 - 146 below).

Reply:
We can't use this function from within `execute_transaction()` because it takes `&self` and this function is mutable

Reply:
Why would we need to call it from within `execute_transaction()`? Once we load something into `self.mast_store`, we don't need to do it again.

Reply:
I completely misunderstood the original comment and intent of this code